sections/builder.js

```diff
@@ -1430,6 +1430,12 @@ export default [
         content: `
           knex('users').count('active as a')
         `
+      },
+      {
+        type: "runnable",
+        content: `
+          knex('users').count(knex.raw('?', ['active']))
```

Comment:
If in these intention is to count column name "active" it should be `knex.raw('??', ['active'])`

Reply:
Ah yes, my intent was to use "active" as an identifier instead of a value.  Both this and https://github.com/tgriesser/knex/pull/2257 are now updated to use `??`.